### ISSUE TITLE
Mobile portait mode header is broken in supplemental guideance

### ISSUE THREAD

**Issue by SteveALee** [w3c/wai-minimal-header-design#95]
The logo header overflows to the right giving 2 scroll bars (wcag fail).

![image](https://user-images.githubusercontent.com/618922/161232976-e27140b6-acdf-483a-a316-39ae35db1af2.png)

Need to move subtitle to be below.

HT to @hidde for reporting

**Comment by shawna-slh** [comment]
Thanks to both! (close issue?)

**Comment by SteveALee** [comment]
Oh, Thought I had.
